pkg/plugin/installer/installer.go

```diff
@@ -75,8 +75,8 @@ func NewForSource(source, version string) (Installer, error) {
 }
 
 // FindSource determines the correct Installer for the given source.
-func FindSource(location string) (Installer, error) {
-	installer, err := existingVCSRepo(location)
+func FindSource(location, version string) (Installer, error) {
```

Comment:
`FindSource` is a public function. Helm follows semantic versioning so the public API can't change in a backwards compatibility breaking way. You'll need to leave the function signature for `FindSource` alone and create a new function with a different signature. You can find more details on this method at https://go.dev/blog/module-compatibility

Reply:
I understand, I will create a new function similiar to it, perhaps `FindSourceWithVersion`